src/VisualStudio/Core/Def/PublicAPI.txt

```diff
@@ -1,6 +1,8 @@
 Microsoft.VisualStudio.LanguageServices.Progression.GraphNodeCreation
 Microsoft.VisualStudio.LanguageServices.VisualStudioWorkspace
 Microsoft.VisualStudio.LanguageServices.VisualStudioWorkspace.CreatePortableExecutableReference(string filePath, Microsoft.CodeAnalysis.MetadataReferenceProperties properties) -> Microsoft.CodeAnalysis.PortableExecutableReference
+Microsoft.VisualStudio.Shell.Interop.IComWrapperFactory
+Microsoft.VisualStudio.Shell.Interop.IComWrapperFactory.CreateAggregatedObject(object managedObject) -> object
```

Comment:
Do we believe we'll get a real interop type before we ship some other public bits? Just don't want people taking a dependency on it.